blogs/blocks/footer/footer.css

```diff
@@ -51,4 +51,9 @@ footer #footer {
 
 .keysight-footer #copyright ul li {
   font-size: 14px;
+  color: #524f56;
```

Comment:
I recommend that when you have a var defined for a color in styles.css, that you use them in your block css.
In the future, if this color changes, you'd only have to change it in one place.
In this case and line 58, you should use:
color: var(--color-gray-500);